service/history/timerBuilder.go

```diff
@@ -280,17 +289,21 @@ func (tb *timerBuilder) loadActivityTimers(msBuilder *mutableStateBuilder) {
 					EventID:     v.StartedID,
 					TimeoutType: w.TimeoutType_START_TO_CLOSE,
 					TimeoutSec:  v.StartToCloseTimeout,
-					TaskCreated: v.TimerTaskStatus == TimerTaskStatusCreated}
+					TaskCreated: (v.TimerTaskStatus & TimerTaskStatusCreatedStartToClose) != 0}
```

Comment:
ah.. the serialized mutable state has one single TimerStatus per activity. Makes sense now. 
As for the Equals(), its ok if you don't want to do it. It helps with future refactoring. also its somewhat easy to get wrong, so having it in one place make sense. Anyways, not critical.